.platform/confighooks/predeploy/predeploy.sh

```diff
@@ -0,0 +1,16 @@
+# START_FEATURE elastic_beanstalk
+#!/bin/bash
+source $PYTHONPATH/activate
+
+# START_FEATURE django_react
+npm install -G nwb
+npm install
+./node_modules/.bin/nwb build --no-vendor
+# END_FEATURE django_react
```

Comment:
Since we're referencing `nwb` on line 8 via the local node modules path, we can get rid of the global install.

```
# START_FEATURE django_react
npm install
./node_modules/.bin/nwb build --no-vendor
# END_FEATURE django_react